FirebaseRemoteConfigSwift/Tests/SwiftAPI/PropertyWrapperTests.swift

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseRemoteConfig
+import FirebaseRemoteConfigSwift
+import XCTest
+
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+  @available(iOS 14.0, macOS 11.0, macCatalyst 14.0, tvOS 14.0, watchOS 7.0, *)
+  class PropertyWrapperTests: APITestBase {
+    // MARK: - Test fetching Remote Config JSON values into struct property
+
+    struct Recipe: Decodable {
+      var recipeName: String
+      var ingredients: [String]
+      var cookTime: Int
+    }
+
+    static let fallbackString = "fallback"
+    static let fallbackInt = 50
+    static let fallbackFloat: Float = 50.2
+    static let fallbackDouble: Double = 16_777_216.333921
+    static let fallbackDecimal: Decimal = 235
+    static let fallbackData = "hello".data(using: .utf8)!
+    static let fallbackArray = ["mango", "pineapple", "papaya"]
+    static let fallbackDict = [
+      "session 0": "breakfast", "session 1": "keynote", "session 2": "state of union",
+    ]
+    static let fallbackJSON = Recipe(
+      recipeName: "muffin", ingredients: ["flour", "sugar"], cookTime: 45
+    )
+
+    struct PropertyWrapperTester {
+      @RemoteConfigProperty(key: Constants.stringKey, fallback: "")
```

Comment:
I would avoid re-using `Constants` in this file as well.

Reply:
I've excluded this test from API tests because this one does need fetch and activate from console and current API test seem cache config results form other tests.

Feel free to kick off a new PR if you find a solution to fix this.